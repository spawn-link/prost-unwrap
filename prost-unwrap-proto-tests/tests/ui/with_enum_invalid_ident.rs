mod original {}
mod test {
    // NB: trybuild alters the cwd, so the adjustment is needed; actual cwd
    // within trybuild test is
    // $WORKSPACE/target/tests/trybuild/prost-unwrap-proto-tests
    prost_unwrap::include!(from_source(
        root::inner,
        "../../../../prost-unwrap-proto-tests/.proto_out/root.inner.rs"
    )
    .with_original_mod(crate::original)
    .with_this_mod(crate::test)
    .with_enum("B"));
}

fn main() {}
