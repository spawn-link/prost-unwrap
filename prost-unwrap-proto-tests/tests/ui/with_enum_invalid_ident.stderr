error: Argument must be a path literal relative to `with_original_mod` argument, e.g. `root::Something`
  --> tests/ui/with_enum_invalid_ident.rs:11:16
   |
11 |     .with_enum("B"));
   |                ^^^
